quiz.rb

```diff
@@ -2,16 +2,47 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
+class Dog
+ def initialize(name,breed,age)
+  @name = name
+  @breed = breed
+  @age = age
+     attr_reader :name
+     attr_reader :breed
+     attr_reader :age
+   end
+
+
 # Question 2
 # Create a new Dog object.
 
+def treats
+        puts "loves bones"
+    end
+
```

Comment:
Just like in JavaScript, we use a method/function to create new objects. In JavaScript, we might write
`var dog = new Dog("sparky", "dalmation", 7);`
In Ruby, we might write
`dog = Dog.new("sparky", "dalmation", 7)`